### ISSUE TITLE
eslint and prettier clash over aligning long a ? b : c statement with 4 space tabs

### ISSUE THREAD

**Issue by jlchereau** [eslint/eslint#10930]
For example
![image](https://user-images.githubusercontent.com/2556751/46573009-07987500-c987-11e8-9191-301f543ffa31.png)
but with 28 spaces:
![image](https://user-images.githubusercontent.com/2556751/46573012-1b43db80-c987-11e8-943d-0ae8391ae34e.png)

**Tell us about your environment**

* **ESLint Version:** 5.6.1
* **Prettier Version:** 1.14.3
* **Node Version:** 10.11.0
* **npm Version:** 6.4.1

**Please show your full configuration:**

```json
{
    "env": {
        "node": true,
        "es6": true
    },
    "extends": [
        "airbnb-base",
        "plugin:node/recommended",
        "prettier"
    ],
    "parser": "esprima",
    "plugins": [
        "node",
        "prettier"
    ],
    "root": true,
    "rules": {
        "indent": ["error", 4, { "SwitchCase": 1 }],
        "no-plusplus": ["error", { "allowForLoopAfterthoughts": true }],
        "no-underscore-dangle": 0,
        "node/exports-style": ["error", "module.exports"],
        "prettier/prettier": ["error", {
            "singleQuote": true,
            "tabWidth": 4
        }]
    }
}
```

And for prettier
```
{
    "singleQuote": true,
    "tabWidth": 4
}
```
</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

Any code that has a long a ? b : c statement will trigger the error

```js
            new PageComponent({
                tool: 'textbox',
                top: 380,
                left: 80,
                width: 380,
                height: 100,
                properties: {
                    question: options.question,
                    solution: solutions[0],
                    validation:
                        solutions.length > 1
                            ? `// ignoreCaseMatch ${JSON.stringify([
                                `^(?:${escaped.join('|')})$`
                            ])}`
                            : '// ignoreCaseEqual'
                }
            })
```

In WebStorm

![image](https://user-images.githubusercontent.com/2556751/46573088-0287f580-c988-11e8-849b-24cb7db185d3.png)

But also using grunt-eslint configured in grunfile.js as follows:

```
        eslint: {
            files: [
                '*.js',
                'templates/*.es6',
                'src/js/**/*.es6',
                'test/**/*.es6'
            ],
            options: {
                config: '.eslintrc'
            }
        },
```

**What did you expect to happen?**

Fix with eslint should find an alignment that does not trigger any any error.

**What actually happened? Please include the actual, raw output from ESLint.**

See screenshots above.

**Comment by kaicataldo** [comment]
@jlchereau ESLint's `indent` rule and Prettier's indentation styles do not match - they're completely separate implementations and are two different approaches to solving the same problem ("how do we enforce consistent indentation in a project").

When using Prettier, you shouldn't be using ESLint's `indent` rule at all. In your current configuration, the `prettier` config disable `indent` and then you're turning it back on using `"indent": ["error", 4, { "SwitchCase": 1 }],` int the `rules` property of your config. I don't use `eslint-plugin-prettier`, but I believe you should be letting that rule handle any warnings about indentation.

**Comment by nzakas** [comment]
It looks like the conversation is stalled here. As this is a question rather than an action item, I'm closing the issue. If you still need help, please send a message to our [mailing list](https://groups.google.com/group/eslint) or [chatroom](https://gitter.im/eslint/eslint). Thanks!
